here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/util/storage/RequestHelper.java

```diff
@@ -134,6 +134,35 @@ public class RequestHelper {
     return request;
   }
 
+  /**
+   * Helper method to create WriteFeatures request for deleting multiple features.
+   *
+   * @param collectionName name of the storage collection
+   * @param features       feature object array to be deleted
+   * @return WriteFeatures request that can be used against IStorage methods
+   */
+  public static @NotNull WriteXyzFeatures deleteFeaturesRequest(
+      final @NotNull String collectionName, final @NotNull List<String> features) {
+    final WriteXyzFeatures request = new WriteXyzFeatures(collectionName);
+    for (String feature : features) {
+      request.delete(feature, null);
+    }
+    return request;
+  }
+
+  /**
+   * Helper method to create WriteFeatures request for deleting given feature.
+   *
+   * @param collectionName name of the storage collection
+   * @param feature        feature object to be deleted
+   * @return WriteFeatures request that can be used against IStorage methods
+   */
+  public static @NotNull WriteXyzFeatures deleteFeatureRequest(
```

Comment:
As we are only passing Id and not entire feature, I suggest renaming this method to `deleteFeatureByIdRequest()` or something similar